documentation/slate/source/includes/apis.md

```diff
@@ -83,8 +78,17 @@ accessible local mobility services:
         {"href": "https://api.navitia.io/v1/coverage/fr-idf/"}
     ]
 }
+
 ```
 
+Also known as `/coord` service.
```

Comment:
From official documentation https://github.com/tripit/slate/wiki/Markdown-Syntax
"Code samples will appear in the dark area to the right of the main text. We recommend positioning code samples right under headers in your markdown file."
This sentence is so the first of the chapter. Above, it is the shell example